rpc/src/client/bin/main.rs

```diff
@@ -488,6 +492,7 @@ async fn recv_events(mut subs: Subscription, max_events: Option<u32>) -> Result<
     let mut event_count = 0u64;
     while let Some(result) = subs.next().await {
         let event = result?;
+        let event: DialectEvent<<LatestDialect as Dialect>::Event> = event.into();
```

Comment:
Shall we discover the compat and then use the appropriate dialect here?
I don't who is or has ever used this rpc binary, but if someone is they may be surprised if they try running it against a 0.34 node. Or maybe it'll just work since one can decode a 0.37 event using the 0.34 schema (but not the other way around).

Reply:
This is only used to display the events by converting them back into JSON. I think it's actually better to show events in plain text even if the original JSON is from 0.34.

That said, this binary does not do version discovery at all now, so the clients would always decode as 0.37. This we shall change.

Reply:
Added version discovery for the CLI in https://github.com/informalsystems/tendermint-rs/pull/1193/commits/2d4a3336458f3d0cea6ae90928ab3f2aabdfb4ef